<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zCreateCurve</class>
 <widget class="QDialog" name="zCreateCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create Curve</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="mainBox" native="true">
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QToolButton" name="createNodeButton">
          <property name="text">
           <string>Create Curve</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="nodeDropdown">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="selectNodeButton">
          <property name="toolTip">
           <string>Select node</string>
          </property>
          <property name="text">
           <string>S</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="addToCurveButton">
          <property name="text">
           <string>Add Transforms</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeFromCurveButton">
          <property name="text">
           <string>Remove Transforms</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ListWidgetWithDrop" name="transformList">
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListWidgetWithDrop</class>
   <extends>QListWidget</extends>
   <header>zMayaTools/qt_widgets/list_widget_with_drop.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qt_resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
